system/scripts/zone/core/normal_tx.cs

```diff
@@ -185,4 +187,87 @@ public NormalTxResult SCR_TX_SKILL_UP(Character character, int[] numArgs)
 
 		return NormalTxResult.Okay;
 	}
+
+	[ScriptableFunction]
+	public NormalTxResult SCR_TX_EQUIP_CARD_SLOT(Character character, string strArg)
+	{
+		var argList = strArg.Split('#');
+		if (argList.Length < 2)
+		{
+			Log.Warning("SCR_TX_EQUIP_CARD_SLOT: User '{0}' failed to parse at least 2 items {1}.", character.Username, argList);
+			return NormalTxResult.Fail;
+		}
+		if (!int.TryParse(argList[0], out var slot))
+		{
+			Log.Warning("SCR_TX_EQUIP_CARD_SLOT: User '{0}' failed to parse slot {1}.", character.Username, argList[0]);
+			return NormalTxResult.Fail;
+		}
+		// Lua to C# 0 -> 1 based array offset
+		slot += 1;
+		if (!long.TryParse(argList[1], out var itemWorldId))
+		{
+			Log.Warning("SCR_TX_EQUIP_CARD_SLOT: User '{0}' failed to parse slot {1}.", character.Username, argList[1]);
+			return NormalTxResult.Fail;
+		}
+
+		if (!character.Inventory.TryGetItem(itemWorldId, out var card))
+		{
+			Log.Warning("SCR_TX_EQUIP_CARD_SLOT: User '{0}' no card found {1}.", character.Username, argList[0]);
+			return NormalTxResult.Fail;
+		}
+
+		if (character.Inventory.EquipCard(slot, itemWorldId) != InventoryResult.Success)
```

Comment:
Unless I missed it, you don't seem to have a sanity check for the slot, right? This means users could technically send any slot id and equip `int.MaxValue` number of cards.

Reply:
I see what you mean I'll add a slot check within EquipCard.
`
if (slot < 1 && slot > 15)
	return InventoryResult.InvalidSlot;
`
Interestingly UnequipCard had the check, so 🤷 guess I forgot to add the check here.